build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -29,6 +29,9 @@
     tf_models.EFFICIENTNET_V2_S_FP32,
     tf_models.MINILM_L12_H384_UNCASED_INT32_SEQLEN128,
     tf_models.RESNET50_TF_FP32,
+    torch_models.MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH,
+    # Disabled due to https://github.com/openxla/iree/issues/11447.
+    #torch_models.MODEL_UNET_2D_FP32_TORCH,
```

Comment:
Are we tracking any "small" PyTorch models? It would be nice to have coverage for a variety of model architectures

Reply:
We'll be adding more PyTorch models in the coming weeks. I'll keep this in mind and make sure we add small torch models (probably an EfficientNet small since we have a TF version of this).